### ISSUE TITLE
Getting different accuracy in different runs for text classification in auto-keras

### ISSUE THREAD

**Issue by priyaJagan90** [keras-team/autokeras#643]
I am getting different accuracy while building the text classification model each time in auto-keras for the same training and validation set. I wanted to know the reason why does it give different accuracy at different runs though training and validation set is same for each run.

**Comment by jetjodh** [comment]
The seed for the weight initialization is set randomly each time you run the model. For reproducible results, you need the set the seed at the beginning of the program.
For example,
`np.seed(2019)`

**Comment by priyaJagan90** [comment]
Thank you. I will try this.
